docs/design/rosbag2_record_replay_service.md

```diff
@@ -22,23 +22,50 @@ Service introspection is saved like general topic. But they should use different
   
   [NameSpace/]_TopicName
 
-- Service introspection topic
+- Service event topic
   
   [NameSpace/]ServiceName/_service_event
 
-So if the string ends with `/_service_event` in topic name, it should be service introspection topic. Of course, it is possible that hidden topic has the same name. So message type is used for further check. The message type for service introspection must ends with "_Event" (e.g. example_interfaces/srv/AddTwoInts_Event). Therefore, service introspection topic can be identified.
+The following criteria will be used to determine a service event topic:
+1. The topic ends with `/_service_event`
+2. The topic type name matches  `*/srv/*_Event` (e.g. `example_interfaces/srv/AddTwoInts_Event`)
 
 ### Expand the 'record' command
 
-Add 1 parameter.
+Add 3 parameters.
 
-- `-S [SerivceName1 ServiceName2 ...]` or `--services [SerivceName1 ServiceName2 ...]`
+- `-S ServiceName1 [ServiceName2 ...]` or `--services ServiceName1 [ServiceName2 ...]`
   
-    If the user doesn't specify service names, all service introspection topics are recorded.
+    Specify which services will be recorded. User must specify at least one service.
+
+- `--all-topics`
+
+    Record all topics. Hidden topic are excluded (include service event topics).
```

Comment:
It doesn't make sense to include service event topics in `--all-topics` if we will have `--all` command which will be doing exactly the same.
It will be more reasonable to have `--all-topics` options when only regular topics without service events would be recorded. 
I was under the impression that we clarified this case in some of the comments above and after the last WG meeting.

Reply:
After some consideration, it seems I misunderstood the phrase 
> Hidden topic are excluded (include service event topic)  

To avoid confusion for others I would suggest rewording it to be more clear. 
Something like. **"Hidden topics including service event topics are excluded."**

Reply:
> To avoid confusion for others I would suggest rewording it to be more clear.
> Something like. "Hidden topics, including service event topics are excluded."

Sorry to make you confused.  
I will update description as your suggestion.